llvm/lib/SYCLLowerIR/RenameKernelSYCLNativeCPU.cpp

```diff
@@ -45,5 +45,18 @@ RenameKernelSYCLNativeCPUPass::run(Module &M, ModuleAnalysisManager &MAM) {
     }
   }
 
+  //<<<<<<< HEAD
+  //  for (auto &F : CalledSet) {
+  //    auto NewName = sycl::utils::addSYCLNativeCPUSuffix(F->getName());
+  //    F->setName(NewName);
+  //    auto Comdat = F->getComdat();
+  //    if (Comdat) {
+  //      auto NewComdat = M.getOrInsertComdat(NewName.str());
+  //      F->setComdat(NewComdat);
+  //    }
+  //    ModuleChanged |= true;
+  //  }
+  //=======
+  //>>>>>>> sycl
```

Comment:
A merge gone wrong?

Reply:
Yes :) That's fixed now, thanks for spotting it